drivers/Web/config.py

```diff
@@ -12,7 +12,7 @@
 
 class config:
     def run_ui(self):
-        conn_pool = postgresql_connection_pool(5)
+        conn_pool = postgresql_connection_pool(max_connections=5)
```

Comment:
this bug indicate that the constructor of postgresql_connection_pool is not typed enough

Reply:
Can i type keyword arguments in python?

Reply:
yes